portal/lib/patient_groups.php

```diff
@@ -0,0 +1,562 @@
+<?php
+
+/**
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2022 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once('./../../interface/globals.php');
+require_once("$srcdir/patient.inc");
+
+use OpenEMR\Core\Header;
+use OpenEMR\Services\DocumentTemplates\DocumentTemplateService;
+
+$templateService = new DocumentTemplateService();
+$group_list =  $templateService->fetchDefaultGroups();
+$profile_list = $templateService->fetchDefaultProfiles();
+
+if ($_POST['mode'] === 'fetch_groups') {
+    $rtn = $templateService->getPatientsByGroup($group_list);
+    if ($rtn) {
+        echo $rtn;
```

Comment:
initially seemed odd since not escaped, but looks like this function returns an array, so not sure how can echo that

Reply:
yeah, abandoned code and forgot to remove. never called from anywhere.